### ISSUE TITLE
Dataparallel issue

### ISSUE THREAD

**Issue by mpicci** [vikasverma1077/manifold_mixup#4]
Hi,
in manifold_mixup_hidden_ssl.py row 215.
Using two GPU with DataParallel the dim of lam is two, .item() cannot convert to one single scalar.

**Comment by vikasverma1077** [comment]
@mpicci Hi there! For SSL, please check our new detailed work here https://github.com/vikasverma1077/ICT

It uses single GPU for sake of simplicity.

**Comment by jimmykobe** [comment]
Yeah, I have the same problem as @mpicci. I have noticed that the implementation has commented the dataparallel out.
